csharp/Adapter/Microsoft.Spark.CSharp/Interop/Ipc/SerDe.cs

```diff
@@ -174,6 +179,20 @@ public static void Write(Stream s, byte value)
             totalWriteNum += 1;
         }
 
+        public static void WriteBytes(Stream s, byte[] value)
```

Comment:
This method could be confused with Write(Stream, byte[]) method below. Why is this necessary? Can we give a better name?